### ISSUE TITLE
Ensure all colors are color-blind safe

### ISSUE THREAD

**Issue by rdmurphy** [datadesk/datawrapper-migration#8]
The main thing I saw is our link style in the source section. I'm not sure where the light blue originated from, but it doesn't have sufficient contrast to be [WCAG 2.0 compliant](https://www.w3.org/WAI/). We should probably do this check with **all** colors we're adding here. (If we haven't already!)
